src/services/api/graphql/generate/schema_from_composite.py

```diff
@@ -29,50 +29,23 @@
     sys.path.append("/usr/libexec/vyos/services/api")
```

Comment:
I think we should use global constants rather than duplicate hardcoded paths in files.

Reply:
Quite right, thanks. Path from defaults.py now used in all instances.